site-templates/pages/versioning.gomd

```diff
@@ -6,7 +6,7 @@
             <td style="padding:15px">   
                     <b><center>NOTICE</center>
                         <br />
-                        The support for API version <span class="pre">/v3/</span> is now released. This change went into effect with release version 21.03.1, on MM DD, YYYY. 
+                        The support for API version <span class="pre">/v3/</span> is now available. This change went into effect with release version 21.02.1, on Feb 12th, 2021. 
```

Comment:
Weren't we going to add "v3 beta" to this versioning page? Might need to add it to other pages, as necessary

Reply:
oops, keep forgetting that